ale/drivers/ody_drivers.py

```diff
@@ -0,0 +1,123 @@
+import spiceypy as spice
+
+import ale
+from ale.base.data_naif import NaifSpice
+from ale.base.label_isis import IsisLabel
+from ale.base.type_sensor import LineScanner
+from ale.base.base import Driver
+
+import pvl
+
+class OdyThemisIrIsisLabelNaifSpiceDriver(LineScanner, IsisLabel, NaifSpice, Driver):
+
+    @property
+    def instrument_id(self):
+        inst_id = super().instrument_id
+
+        if inst_id not in ["THEMIS_IR"]:
+            raise Exception(f"{inst_id} is not a valid THEMIS instrument name. Expecting THEMIS_IR")
+
+        return inst_id
+
+    @property
+    def sensor_model_version(self):
+        return 1
+
+    @property
+    def spacecraft_name(self):
+        name = super().spacecraft_name.replace('_', ' ')
+        if name != "MARS ODYSSEY":
+            raise Exception("{name} for label is not a valid Mars Odyssey spacecraft name")
+        return name
+
+    @property
+    def ikid(self):
+        return self.label['IsisCube']['Kernels']['NaifFrameCode']
+
+    @property
+    def line_exposure_duration(self):
+        return (33.2871/1000 * self.line_summing)
```

Comment:
May want to put doc strings on these explaining the hard coded values.

Reply:
I could I guess but it would just be "look at ISIS camera models", the model doesn't describe where the numbers came from.

Reply:
Pointing to the file would probably be fine